adafruit_circuitplayground/circuit_playground_base.py

```diff
@@ -184,6 +184,58 @@ def detect_taps(self, value):
                     value, 60, time_limit=10, time_latency=50, time_window=255
                 )
 
+    # pylint: disable-msg=too-many-arguments
+    # Many default arguments which will probably not need changing in most cases
+    def configure_taps(
+        self,
+        value,
```

Comment:
`value` is unclear. I had to go to the LIS3DH library and look up what arguments `set_tap` took to figure out what this was looking for. It could be called `tap` like it is in `set_tap`, which I feel would clarify it some, however the requested documentation will really be the thing that will resolve this issue.

Reply:
I stole the `value` from the `detect_taps()` setter, but I agree, `tap` is a better name. Changed in the next commit.